integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwSample.java

```diff
@@ -0,0 +1,473 @@
+    createSecretForBaseImages(domainNamespace);
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, domainNamespace);
+  }
+
+  /**
+   * Test JRF domain on pv samples using domains created by wlst.
+   * Verify Pod is ready and service exists for both admin server and managed servers.
+   * Verify EM console is accessible.
+   */
+  @Test
+  @DisplayName("Test JRF domain on PV sample")
+  public void testJrfDomainInPvUsingWlst() {
+    setupSample();
+    // create persistent volume and persistent volume claims used by the samples
+    createPvPvc(domainUid);
+
+    //create WebLogic secrets for the domain
+    createSecretWithUsernamePassword(domainUid + "-weblogic-credentials", domainNamespace,
+            ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
+    // create RCU credential secret
+    createRcuSecretWithUsernamePassword(domainUid + "-rcu-credentials", domainNamespace,
+        RCUSCHEMAUSERNAME, RCUSCHEMAPASSWORD, RCUSYSUSERNAME, RCUSYSPASSWORD);
+
+    Path sampleBase = Paths.get(tempSamplePath.toString(),
+        "scripts/create-fmw-infrastructure-domain/domain-home-on-pv");
+
+    //update create-domain-inputs.yaml with the values from this test
+    updateDomainInputsFile(domainUid, sampleBase);
+
+    // change image name
+    assertDoesNotThrow(() -> {
+      replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString(),
+              "image: container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4",
+              "image: " + FMWINFRA_IMAGE_TO_USE_IN_SPEC);
+    });
+
+    // run create-domain.sh to create domain.yaml file, run kubectl to create the domain and verify
+    //verify EM console is accessible
+    createDomainAndVerify(domainUid, sampleBase);
+
+  }
+
+  private void createDomainAndVerify(String domainName, Path sampleBase) {
+
+    // run create-domain.sh to create domain.yaml file
+    CommandParams params = new CommandParams().defaults();
+    params.command("sh "
+            + Paths.get(sampleBase.toString(), "create-domain.sh").toString()
+            + " -i " + Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString()
+            + " -o "
+            + Paths.get(sampleBase.toString()));
+
+    logger.info("Run create-domain.sh to create domain.yaml file");
+    boolean result = Command.withParams(params).execute();
+    assertTrue(result, "Failed to create domain.yaml");
+
+    // run kubectl to create the domain
+    logger.info("Run kubectl to create the domain");
+    params = new CommandParams().defaults();
+    params.command("kubectl apply -f "
+            + Paths.get(sampleBase.toString(), "weblogic-domains/" + domainName + "/domain.yaml").toString());
+
+    result = Command.withParams(params).execute();
+    assertTrue(result, "Failed to create domain custom resource");
+    logger.info("Run create-domain.sh to create the domain");
```

Comment:
where do we call this script? I don't see it here.

Reply:
Sorry, line195 logging infor is a mistake. I removed it. The correct one is line 176
logger.info("Run create-domain.sh to create domain.yaml file");
    CommandParams params = new CommandParams().defaults();
    params.command("sh "
            + Paths.get(sampleBase.toString(), "create-domain.sh").toString()
            + " -i " + Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString()
            + " -o "
            + Paths.get(sampleBase.toString()));